src/store/localStorage.js

```diff
@@ -0,0 +1,30 @@
+const CONSTS = {
+  configuratorSettings: 'configuratorSettings',
+  configuratorSettingsVersion: 1
+};
+
```

Comment:
I like how super generic this is, but I'm wondering if we should just have wrapped versions of this that we invoke from the users that remove some of the boiler plate

Reply:
mmmm Yeah we might have a factory for the localstorage configuration objects. Which will parse it and make it usable with the current version of the configurator.